io/rtp/src/main/java/org/mobicents/media/server/impl/rtp/channels/MediaChannel.java

```diff
@@ -644,8 +644,27 @@ public void negotiateFormats(MediaDescriptionField media) {
 		
 		// Map payload types tp RTP Format
 		for (int payloadType : media.getPayloadTypes()) {
-			RTPFormat format = AVProfile.getFormat(payloadType, AVProfile.AUDIO);
+			RTPFormat format = null;
+			if (payloadType < 96 )
+				format = AVProfile.getFormat(payloadType, AVProfile.AUDIO);
+			else
+			{
+				// it's a dynamic payload, we need to found them via the name of the codec.
+				
+				String codec = media.getFormat(payloadType).getCodec();
+				logger.debug(" dynamic payload:" + payloadType + " codec:"+codec);
```

Comment:
Protect logging statement: if(logger.isDebugEnabled())
http://stackoverflow.com/questions/6504407/is-there-a-need-to-do-a-iflog-isdebugenabled-check

Also, print the channel's ssrc attribute in the logging statement so we can have proper context.